paasta_tools/utils.py

```diff
@@ -3013,7 +3013,15 @@ def get_pipeline_config(service: str, soa_dir: str = DEFAULT_SOA_DIR) -> List[Di
 def get_pipeline_deploy_groups(
     service: str, soa_dir: str = DEFAULT_SOA_DIR
 ) -> List[str]:
-    pipeline_steps = [step["step"] for step in get_pipeline_config(service, soa_dir)]
+    pipeline_steps = []
+    for step in get_pipeline_config(service, soa_dir):
+        if(step.get("parallel", False)):
+            for parallel_step in step.get("parallel"):
+                if(parallel_step.get("step", False)):
+                    pipeline_steps.append(parallel_step["step"])
```

Comment:
it might be worth adding a comment here about why we're doing this (and maybe what the config looks like for a `parallel` block? could either have a link here to y/deploy.yaml with how parallel works documented there or update https://paasta.readthedocs.io/en/latest/deploy_groups.html to have an example deploy.yaml with a parallel block)

Reply:
Got it. I may need to add it the paasta doc as the y/link wont be available for checking outside of our org. Thanks for the quick review @nemacysts

Reply:
no worries about adding the ylink here - we already reference a bunch of ylinks :p (it is nice to have things documented in the open docs, though)